examples/jenkins/pipeline/pipelinetemplate.json

```diff
@@ -19,6 +19,9 @@
         "creationTimestamp": null,
         "labels": {
           "name": "sample-pipeline"
+        },
+        "annotations": {
+          "downstream.openshift.io/deployments": "[{\"name\": \"frontend\", \"namespace\": \"\"}]"
```

Comment:
Do we want to distinguish between the use the jenkins pipeline has for a
resource?  I.e. if the jenkins pipeline runs an A/B failover flow, would
they point to a route?

On Fri, Jun 10, 2016 at 2:52 PM, Ben Parees <notifications@github.com>
wrote:

> In examples/jenkins/pipeline/pipelinetemplate.json
> <https://github.com/openshift/origin/pull/9275#discussion_r66663632>:
>
> > @@ -19,6 +19,9 @@
> >          "creationTimestamp": null,
> >          "labels": {
> >            "name": "sample-pipeline"
> > +        },
> > +        "annotations": {
> > +          "downstream.openshift.io/deployments": "[{\"name\": \"frontend\", \"namespace\": \"\"}]"
>
> I don't love that it's deployment specific. What's the equivalent
> annotation for builds?
>
> downstream.openshift.io/builds
>
> that was the intent of the generic "downstream.openshift.io" namespace,
> with the /foo being the downstream resource type if we decide to add more.
>
> but i'd be fine with a generic annotation where the the resource type was
> part of the value. @spadgett <https://github.com/spadgett> were just
> throwing something up for discussion.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9275/files/b8845b2e1f6b0e0a3f0bf50d7b5bd0cb9c152a63#r66663632>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p3q1p1xyuCpg4ekLY1ZEmR7IIsCIks5qKbJ6gaJpZM4IzIrf>
> .
>